twoliter/embedded/build.Dockerfile

```diff
@@ -114,18 +52,16 @@ USER builder
 ENV PACKAGE=${PACKAGE} ARCH=${ARCH}
 COPY ./packages/${PACKAGE}/ .
 
-COPY --chown=builder --from=rpm-macros-and-bconds /home/builder/generated.* .
-
-# Merge generated bconds with the package spec, and put sources in the right place. Set Epoch values for the package 
+# Copy over the target-specific macros, and put sources in the right place. Set Epoch values for the package
 # and all of its subpackages. Additionally, for packages declaring explicit package versions requirements via 
 # "{Requires,Conflicts,Obsoletes}: = ...", ensure the epoch for the specified package is set.
 # "[Requires|Conflicts|Obsoletes]:.*[=>,>,<,<=,=]\) \(\%{version}\-\%{release}$\)" matches any line in a form like:
 # "Requires: %{name}-modules = %{version}-%{release}". The full `sed` expression below captures this match into
 # two groups and insert the Epoch value such that the result is "Requires: %{name}-modules = EPOCH:%{version}-%{release}"
 RUN \
-   cat "/usr/lib/rpm/platform/${ARCH}-bottlerocket/macros" generated.rpmmacros > .rpmmacros \
+   cp "/usr/lib/rpm/platform/${ARCH}-bottlerocket/macros" .rpmmacros \
    && echo "Epoch: ${BUILD_EPOCH}" >> rpmbuild/SPECS/${PACKAGE}.spec \
-   && cat generated.bconds ${PACKAGE}.spec >> rpmbuild/SPECS/${PACKAGE}.spec \
+   && cp ${PACKAGE}.spec rpmbuild/SPECS/${PACKAGE}.spec \
```

Comment:
need to reorder this above the previous line, or we'll miss out on the `Epoch` for the top-level package

Reply:
Oh, I see I was blowing it away... this should fix it right? https://github.com/bottlerocket-os/twoliter/compare/224bc50b442bc4f82c3111f955f215d7bc79b195..2721a636b44a41aefdf59a6c9c6c252b1244eb02